portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,40 +36,56 @@
 public class DataServlet extends HttpServlet {
     
 private static final ArrayList<String> messagesList = new ArrayList<String>();
+private static final String index = "/index.html";
+private static final String comment = "Comment";
+private static final String content = "content";
+private static final String textInput = "text-input";
+
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
```

Comment:
Use the variable `comment`.

Reply:
Just changed it for that and a line below